### DISCUSSION TITLE
King Of Fans MR101Z missing MAX setting

### DISCUSSION THREAD

**Discussion by flyize** [zigpy/zigpy#756] [Q&A]
I created an issue a couple of months ago, around the same time that fan settings were migrating to percentages. That has been complete for a while. This issue I posted is stale, and it appears to be in the wrong place. Should I open a new one, or does anyone have any easy ideas I could implement to fix it?

https://github.com/zigpy/zigpy/issues/628

Thanks!

**Comment by Hedda** [comment]
FYI, since part of the same organization on GitHub now members should be able to move issues between all the org repositories.

As such, developers that are members of the organization should be able to move issues from https://github.com/zigpy/zigpy/issues to https://github.com/zigpy/zha-device-handlers/issues and vice versa as needed, like when a uses post issues to the wrong repo.

That way if an issue can now be moved instead, so there should no longer be any need for users to create a copy of an issue.

That is, admins of https://github.com/zigpy organization could move https://github.com/zigpy/zigpy/issues/628 to https://github.com/zigpy/zha-device-handlers/issues
